linkis-commons/linkis-storage/src/main/java/org/apache/linkis/storage/excel/ExcelXlsReader.java

```diff
@@ -258,13 +263,15 @@ public void close() {
       if (fs != null) {
         fs.close();
       }
-      if (inputStream != null) {
-        inputStream.close();
+      try {
+        if (inputStream != null) {
+          inputStream.close();
+        }
+      } catch (IOException e) {
+        logger.info("ExcelXlsReader inputStream closed failed", e);
```

Comment:
move the block up,   if the fs.close() throws exception, the try block will be skipped

Reply:
Separate try catch block of ExeclXlsReader.close() method , so that  if the fs.close() throws exception, the close method of inputStream will continue to execute. Thanks for your good advice.